src/LocationTracker.ts

```diff
@@ -0,0 +1,57 @@
+import Locations, { LocationChange } from './Locations';
+import { SpaceMember } from './Space';
+import { EventListener } from './utilities/EventEmitter';
+
+export type LocationTrackerPredicate<T> = (locationChange: LocationChange<T>) => boolean;
+
+export default class LocationTracker<T> {
+  #locations: Locations;
```

Comment:
TypeScript has support for using the `private` keyword rather than using # to denote a private field

Reply:
I don't think there's really a specific advantage, but we use `private` elsewhere in the library already so we should be consistent

Reply:
OK, let's use private then :+1: